lib/ui/src/components/sidebar/treeview/treeview.js

```diff
@@ -138,12 +138,22 @@ const Tree = props => {
   }
 };
 
+const defaultAlphabeticalSort = (a, b) => (a.id > b.id ? 1 : -1);
```

Comment:
I advise use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) for alpabetical string sort.